### PR TITLE
10203: Update clinical training section content

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As an EDU team member, I need the update content regarding the clinical training program eligibility to the 10203 form and SCO email so that the users are provided with clear, concise and accurate information.
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/25972

related defect https://github.com/department-of-veterans-affairs/va.gov-team/issues/26316
## Testing done
local

## Acceptance criteria
- [x] The "clinical training program" section content in the SCO email below includes the following "header": If the applicant is in a clinical training program for health care professionals

If the student already has a bachelor's or graduate degree and is pursuing a clinical training program, please provide the following:

Has the student earned a bachelor’s or graduate degree in an approved STEM program?
(Programs approved for the Rogers STEM Scholarship)
Name of STEM program of degree conferred:
Is the student currently enrolled in a covered clinical training program for health care professionals?
(Programs approved for the Rogers STEM Scholarship)
Total length of clinical training program (in months or years):
Amount of clinical training program completed to date (in months or years):

- [x] The sub-bullets under the "STEM degree" on the How to Apply page are:

You're enrolled in a bachelor’s degree program for science, technology, engineering, or math (STEM), or
You've already earned a STEM bachelor’s degree and are working toward a teaching certification, or
You've already earned a STEM bachelor's or graduate degree and are pursuing a covered clinical training program for health care professionals.
See eligible degree and clinical training programs

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/edu-benefits/components/EducationWizard.jsx [hunk 1]
```diff
@@ -384,31 +385,60 @@ export default class EducationWizard extends React.Component {
...
+                            You've already earned a STEM bachelor’s degree and
+                            are working toward a teaching certification,{' '}
+                            <b>or</b>
+                          </li>{' '}
+                          <li className="li-styling">
+                            {' '}
+                            You've already earned a STEM bachelor's or graduate
+                            degree and are pursuing a covered clinical training
+                            program for health care professionals. <br />
+                            <a
```
Comment (zurbergram) [inline src/applications/edu-benefits/components/EducationWizard.jsx:431]: Missing onClick

File: src/applications/edu-benefits/components/EducationWizard.jsx [hunk 2]
```diff
@@ -384,31 +385,60 @@ export default class EducationWizard extends React.Component {
...
+                              aria-label="See eligible degree programs, opening in new tab"
```
Comment (zurbergram) [inline src/applications/edu-benefits/components/EducationWizard.jsx:432]: https://github.com/department-of-veterans-affairs/vets-website/pull/17646/files#r655565274

File: src/applications/edu-benefits/components/EducationWizard.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/edu-benefits/components/EducationWizard.jsx:431]: https://github.com/department-of-veterans-affairs/vets-website/pull/17646/files#r655565914

**Review by delli-gatti-michael — APPROVED** [review]

**Pull request merged**